frontend/js/components/environments/Results.js

```diff
@@ -50,21 +62,23 @@ class Results extends Component {
 
                 { bestMatch.map((m) => {
                     return (<BestMatch
-                            key={m.name}
-                            matchText={bestMatch.length > 1 ?
-                              "It's a duplicate!" :
-                              "It's a match!"}
-                            name={m.name}
-                            office={m.office}
-                            matchRate={Math.round(m.matchRate)} />);
+                        key={m.name}
+                        matchText={bestMatch.length > 1 ?
+                            "It's a duplicate!" :
+                            "It's a match!"}
+                        name={m.name}
+                        party="Party Affiliation"
```

Comment:
party-affiliation is dummy data? should be be pulling this from what we pass down.